packages/core/src/routes/sign-in-experience.test.ts

```diff
@@ -25,38 +25,36 @@ describe('signInExperiences routes', () => {
   });
 
   it('PATCH /sign-in-ex/:id', async () => {
-    const companyInfo = {
-      name: 'silverhand',
-      logo: 'http://silverhand.png',
-    };
-    const signInMethods = {
-      primary: ['phone'],
-      secondary: ['email'],
-      disabled: [],
+    const appInfo = {
+      primaryColor: '#000',
+      backgroundColor: '#fff',
+      darkMode: true,
+      darkBackgroundColor: '#000',
+      darkPrimaryColor: '#fff',
+      style: BrandingStyle.Logo,
+      logoUrl: 'http://silverhand.png',
+      slogan: 'silverhand',
     };
 
+    const socialSignInConnectorIds = ['abc', 'def'];
+
     const response = await signInExperienceRequester.patch('/sign-in-ex/default').send({
-      companyInfo,
-      signInMethods,
+      appInfo,
+      socialSignInConnectorIds,
```

Comment:
I was expecting the send input should throw at koaGuard because it seems does not follow the input type guard?

Reply:
just fix related test cases temporarily for now, and will update these cases when updating related APIs next up.

Reply:
Updated 'appInfo' to 'branding'. PTAL

(just fix related test cases temporarily for now, and will update these cases when updating related APIs next up.)